src/compojure/api/middleware.clj

```diff
@@ -171,11 +166,13 @@
    options for the used middlewares (see middlewares for full details on options):
 
    - **:exceptions**                for *compojure.api.middleware/wrap-exceptions*
-       - **:exception-handler**       function to handle uncaught exceptions
-
-   - **:validation-errors**         for *ring.swagger.middleware/wrap-validation-errors*
-       - **:error-handler**           function to handle ring-swagger schema exceptions
-       - **:catch-core-errors?**      whether to catch also `:schema.core/errors`
+       - **:error-handlers**          map of error handlers for different error types.
```

Comment:
`:error-handlers` could be just `:handlers`? as there is already the [Error](http://docs.oracle.com/javase/7/docs/api/java/lang/Error.html), which != Exception.